2-mandatory/2-financial-times.js

```diff
@@ -15,6 +18,13 @@ function potentialHeadlines(allArticleTitles) {
 */
```

Comment:
Nice way to solve this one!
The next step is to think about different arrays that might be passed into this function (in the `allArticleTitles` parameter). What will your code do if `allArticleTitles` only has one title in the array?